index.html

```diff
@@ -1,13 +1,340 @@
+        <div class="gallery">
+          <div class="gallery__photos gallery__photos-left">
+            <img
+            src="./images/photos/1.jpg"
+            alt="palace"
+            class="gallery__img"
+            >
+            <img
+            src="./images/photos/2.jpg"
+            alt="bungalo"
+            class="gallery__img"
+            >
+          </div>
+          <div class="gallery__location">
+            <a
+              href="https://cutt.ly/SyvtCMq"
+              target="blank"
+              class="gallery__link"
+            >
+          Marina Palms
+            </a>
+          / North Miami Beach, FL 33162
+          </div>
+          <div class="gallery__photos">
+            <img
+            src="./images/photos/3.jpg"
+            alt="flag"
+            class="gallery__img"
+            >
+            <img
+            src="./images/photos/4.jpg"
+            alt="hotel"
+            class="gallery__img"
+            >
+          </div>
+          <div class="gallery__location">
+            <a
+            href="https://cutt.ly/7yvtCIz"
+            target="blank"
+            class="gallery__link"
+          >
+          Edition Residens
+            </a>
+          / North Miami Beach, FL 33139
+          </div>
+          <div class="gallery__photos gallery__photos-right">
+            <img
+            src="./images/photos/5.jpg"
+            alt="faena house"
+            class="gallery__img"
+            >
+            <img
+            src="./images/photos/6.jpg"
+            alt="faena beach"
+            class="gallery__img"
+            >
+          </div>
+          <div class="gallery__location">
+            <a
+            href="https://cutt.ly/Tyvt0Ya"
+            target="blank"
+            class="gallery__link"
+          >
+          Faena House
+            </a>
+          / North Miami Beach, FL 33140
+          </div>
+
+
+          <button class="buttom__img">
+            View All
+          </button>
+        </div>
+
+
+      </section>
+      <section class="section">
+        <h3 class="section__title" id="consultation">Consultation</h3>
+        <div class="consultation">
+          <div class="consultation__item">
+            <div class="consultation__description">
+              <h3>How long have you do<br> yourself a gift?</h3>
+              <p>The procedural change mezzo forte starts izoritmichesky pickup at these moments stop LA Mazel and VA
+              Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early
+              twentieth century, microforms, found a distant historical.
+              </p>
+            </div>
+
+
+            <form action="" class="form">
+
+              <input
+                class="form__input"
+                type="text"
+                placeholder="Name"
+                required
+                >
+              <input
+                class="form__input"
+                type="text"
```

Comment:
```suggestion
                type="tel"
                pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}"
```
You could use specific type and add pattern for number validation